Dockerfile

```diff
@@ -379,6 +379,7 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     DEBIAN_FRONTEND=noninteractive sudo apt-get install -y --no-install-recommends \
         php \
         subversion \
+        dotnet6 \
```

Comment:
Please squash the changes to this file into the first commit that initially modified the Docker files.